sql/driver/conn.go

```diff
@@ -141,6 +150,46 @@ func (c *conn) query(stmt parser.Statement, args []driver.Value) (*rows, error)
 	return nil, fmt.Errorf("TODO(pmattis): unimplemented: %T %s", stmt, stmt)
 }
 
+// Send sends the call to the server.
+func (c *conn) Send(call sqlwire.Call) (*rows, error) {
```

Comment:
Oops I changed it on the server end to send(). I'll change that in the next PR. I'm working on moving the rest of the ops to the server presently